### PR TITLE
update build info.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @ansyral @superyyrrzz @hellosnow @qinezh 
#449

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/Incrementals/BuildOutputItem.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System.Collections.Generic;
+
+    public class BuildOutputItem
+    {
+        /// <summary>
+        /// The source file, always form working folder, i.e. start with "~/"
```
Comment (qinezh) [inline src/Microsoft.DocAsCode.Build.Engine/Incrementals/BuildOutputItem.cs]: typo, form=>from

**Comment by ansyral** [comment]
:shipit:

**Comment by qinezh** [comment]
:+1:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**